main.rb

```diff
@@ -1,78 +1,48 @@
-require_relative 'person'
-require_relative 'student'
-require_relative 'teacher'
-require_relative 'capitalize_decorator'
-require_relative 'trimmer_decorator'
-require_relative 'rental'
-require_relative 'book'
-require_relative 'classroom'
+# main = Main.new
+# main.main
```

Comment:
- [ ] Please remove your commented code because it makes your code messy if you want to check your precious code then check your GirHub commit history :slightly_smiling_face: